p2p_wallet/Services/FeatureFlags/FetchesFeatureFlags.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  FetchesFeatureFlags.swift
+//  FeatureFlags
+//
+//  Created by Babich Ivan on 10.06.2022.
+//
+
+import Foundation
+
+public protocol FetchesFeatureFlags {
+    func fetchFeatureFlags(_ completion: @escaping ([FeatureFlag]) -> Void)
```

Comment:
Can you use async/await instead of completion style?

Reply:
too much classes, that contains this closure and I'm going to improve it in Debug View future branch